### PR TITLE
Hueversity turorials assignements PR

### PR DESCRIPTION
**Pull request by omarvides**
Closes #1

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+	path: '/version',
+	handler: function (request, reply) {
+		response = {
+			version: internals.version,
+		}
+		reply(response);
+	}
+});
+
+server.start(function (err) {
```
Comment (AdriVanHoudt) [inline lib/index.js]: new line after function

File: lib/index.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+	Hoek.assert(!err, err);
+	console.log('Hapi server started!', server.info.uri);
+});
```
Comment (AdriVanHoudt) [inline lib/index.js]: new line at the end of the file

File: lib/index.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+server.connection({ port: 8000 });
+
+server.route({
+	method: 'GET',
+	path: '/version',
+	handler: function (request, reply) {
+		response = {
+			version: internals.version,
+		}
+		reply(response);
```
Comment (AdriVanHoudt) [inline lib/index.js]: `return reply()` is better

**Comment by omarvides** [comment]
@AdriVanHoudt  Many thanks!

**Comment by AdriVanHoudt** [comment]
@omarvides np

File: lib/index.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+var Hoek = require('hoek');
+var internals = require('../package.json');
+
+var server = new Hapi.Server();
+
+server.connection({ port: 8000 });
+
+server.route({
+
+	method: 'GET',
```
Comment (hueniverse) [inline lib/index.js]: Indentation should be 4 spaces.

**Pull request closed without merging**